code/modules/power/lighting.dm

```diff
@@ -395,15 +395,13 @@
 
 /obj/machinery/light/ex_act(severity)
 	switch(severity)
-		if(1)
-			qdel(src)
-			return
 		if(2)
 			if (prob(75))
 				broken()
 		if(3)
 			if (prob(50))
 				broken()
```

Comment:
Doesn't really make sense: at 2 it breaks, but at 1 nothing happens?

Reply:
They call parent at the end, broken() is lightswitch-specific thing to break bulb